cluster/util.py

```diff
@@ -130,3 +132,38 @@ def centroid(data, method=median):
     for i in range(len(data[0])):
         out.append(method([x[i] for x in data]))
     return tuple(out)
+
+def HDcentroid(data):
+    dict_words={}
+    dict_weight={}
+    words_per_user=10 #10 words per user. This value is not used.
+    num_users_cluster=len(data)# len(data) is the number of users (user=item)
+
+    for i in range (num_users_cluster):
+        words_per_user=len(data[i])/2 #each profile have 10 pairs of keyword, weight
+        for j in range (words_per_user):
+            word=(data[i])[j*2]
+            if (dict_words.has_key(word)) :
+                dict_words[word]+=1
+                dict_weight[word]+=data[i][2*j+1]
+            else :
+                dict_words[word]=1
+                dict_weight[word]=data[i][2*j+1]
+    #l is a ordered list of the keywords, with the sum of the weight of every popular keyword
+    l=dict_words.items()
+    l.sort(key=lambda x:10000000-x[1])
```

Comment:
Why exactly has the value `1000000` been used here? This line need a small comment explaining what's happening here, and why `1000000`.